plone/app/upgrade/v51/betas.py

```diff
@@ -155,12 +155,16 @@ def reindex_mime_type(context):
     cnt = 0
     for brain in zcatalog.unrestrictedSearchResults(object_provides=ifaces):
         obj = brain._unrestrictedGetObject()
```

Comment:
I think in an upgrade this is fine to not run into permission problems, but this is just a guess.